src/App.tsx

```diff
@@ -1,17 +1,103 @@
-import React from 'react';
-import './App.css';
-
+import React, { useState } from 'react';
 import users from './api/users';
+import './App.css';
+import TodoList from './components/TodoList/TodoList';
+import { preparedTodos } from './types/Preraredtodo';
 
 const App: React.FC = () => {
+  const [title, SetTitle] = useState('');
+  const [user, SetUser] = useState('');
+  const [isValidTitle, SetValidTitle] = useState('');
+  const [isValidUser, SetValidUser] = useState('');
+
+  const handelClick = () => {
+    if (!title) {
+      SetValidTitle('Please enter the title');
+    }
+
+    if (!user) {
+      SetValidUser('Please choose a user');
+    }
+
+    if (!title && !user) {
```

Comment:
actually, my mistake, you should invert this check
```suggestion
    if (title && user) {
```